src/Common/CommandArguments.php

```diff
@@ -15,7 +15,35 @@ trait CommandArguments
     protected $arguments = '';
 
     /**
-     * Pass argument to executable. Its value will be automatically escaped.
+     * @var array
+     */
+    protected $argumentsEnv = [];
+
+
+    /**
+     * Adds an argument and its placeholder value, to be executed
+     *
+     * @param string $argument
+     * @param string|null $key
+     * @param string|null $prefix
+     * @param string|null $separator
+     * @return void
+     */
+    public function addArgument($argument, $key = null, $prefix = null, $separator = ' ')
```

Comment:
Is there a use-case for calling this directly, or could it be made protected?

Reply:
It's a mistake, it should be protected. I'll push a fix.

Reply:
Fic pushed